### ISSUE TITLE
Embed in div element

### ISSUE THREAD

**Issue by finzaiko** [devforth/painterro#84]
How can I embed a div element with specific width height ?
```
<div id="test" style="width:100px;height:100px">
</div>

Painterro({
	id: "test"
}).show()
```
Seem doesn't work
https://jsfiddle.net/7ruofsqm/1/

**Comment by TommyDufmats** [comment]
+1

**Comment by stellarwobble** [comment]
+1

**Comment by ivictbor** [comment]
@finzaiko @TommyDufmats @stellarwobble here how to do it: https://jsfiddle.net/b9hyk7zt/2/
But still here is one small bug I think - buttons bar should have ability to scroll by mouse if it does not fit

![image](https://user-images.githubusercontent.com/1838656/92306660-8a724200-ef99-11ea-80bb-e8b48e0cb350.png)

@AlonaRadchenko I remember that I've included this behaviour when I codded it could you please take a look and fix

**Comment by ivictbor** [comment]
Scroll issue also resolved in `v1.0.36`, here is updated fiddle also https://jsfiddle.net/pvcod3z8/
